cli/ballerina-cli/src/test/java/io/ballerina/cli/cmd/BuildNativeImageCommandTest.java

```diff
@@ -91,12 +106,28 @@ public void testBuildBalFileWithNativeImage() {
 
         System.setProperty(ProjectConstants.USER_DIR, this.testResources.resolve("valid-test-bal-file").toString());
         // set valid source root
-        BuildCommand buildCommand = new BuildCommand(projectPath, printStream, printStream, false, false, true);
+        BuildCommand buildCommand = new BuildCommand(projectPath, printStream, printStream, false, false, true, "");
         // name of the file as argument
         new CommandLine(buildCommand).parse(projectPath.toString());
         buildCommand.execute();
 
         Assert.assertTrue(projectPath.resolve("target").resolve("native").resolve("winery").toFile().exists());
     }
 
+    @Test(description = "Test valid ballerina file with graalvm additional args", enabled = false)
+    public void testBuildBalFileWithAdditionalArgs() throws IOException {
```

Comment:
We don't need to support graalvm image generation for standalone files. We can remove this test case and add validatiosn in build and test commands to fail with an error message if the `--graalvm` flag is passed with a standalone file.